users-react-app/src/pages/Layout.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import Header from '../components/Header/Header';
+
+function Layout(props){
+    return (
+        <React.Fragment>
```

Comment:
This should work too and it´s less verbose 😄 
```suggestion
        <>
```